src/utils/triton.h

```diff
@@ -135,6 +135,10 @@ class TritonError : public UniqueHandle<TRITONSERVER_Error *, TritonError>, publ
     return TritonError(err);
   }
 
+  static TritonError Success() {
+    return TritonError(nullptr);
+  }
+
```

Comment:
I believe this looks fishy. I get, that `TRITONSERVER_Error == nullptr` denotes no error, but `TritonError(nullptr)` is still an `std::exception` object, which suggests an error.

Reply:
True. I removed this Success method. Instead I changed the SendResponse to not require providing the error if it's supposed to be success (nullptr)